pom.xml

```diff
@@ -1,4 +1,4 @@
-<!--
+    <!--
```

Comment:
Nit: probably not necessary?

Reply:
Thanks for the review, I'll fix it later👍🏻